### PR TITLE
Aa/coveralls

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Comment by kshyatt** [comment]
Is this the one I want to merge?

**Comment by kshyatt** [comment]
Can be done later but we should really change all those printlns to use the base logging API

File: src/coveralls.jl [hunk 1]
```diff
@@ -11,6 +11,7 @@ module Coveralls
     using HTTP
     using JSON
     using Compat
+    using MbedTLS
```
Comment (tkelman) [inline src/coveralls.jl]: this needs to be in require, don't import packages that you don't directly depend on

Comment (kshyatt) [reply src/coveralls.jl]: Good catch, sorry

**Pull request merged**